### DISCUSSION TITLE
how to use as(aliases) on join query builder rakit?

### DISCUSSION THREAD

**Discussion by wznuhidayat** [esyede/rakit#38] [Diskusi Umum]
why i cannot user ->join('table_a as a', ......)?

**Comment by esyede** [comment]
Hi, 
The `join()` method implementation are indeed super simple so I doubt it will work for your need. Please utilize the  `to_sql()` method to see if the produced sql was correct.

If not, you can use the raw query to achive this.
https://rakit.esyede.my.id/docs/id/database/raw
